syntax = "proto3";

package student;

service StudentService {
  rpc SubmitStudent(Student) returns (SubmitResponse);
}

message Student {
  string id = 1;
  string first_name = 2;
  string last_name = 3;
  string email = 4;
}

message SubmitResponse {
  string status = 1;
  string message = 2;
}